R/create_config.R

```diff
@@ -0,0 +1,72 @@
+#' Create a list of configuration settings for some details of [sim_linelist()]
+#'
+#' @details The `config` argument in [sim_linelist()] controls the small details
+#' around time windows around infections (time of first contact and last
+#' contact with infector), and the distribution of the Ct value for confirmed
+#' cases.
+#'
+#' Accepted arguments and their defaults are:
+#' * `last_contact_distribution = "pois"`
+#' * `last_contact_distribution_params = c(lambda = 3)`
+#' * `first_contact_distribution = "pois"`
+#' * `first_contact_distribution_params = c(lambda = 3)`
+#' * `ct_distribution = "norm"`
+#' * `ct_distribution_params = c(mean = 25, sd = 2)`
+#' * `network = "adjusted"`
```

Comment:
While not expected to be changed, it would be worth documenting what the accepted values of each are, e.g. `last_contact_distr` must be a quoted distribution name

Reply:
Resolved in #84.